activejob/test/cases/railtie_test.rb

```diff
@@ -0,0 +1,37 @@
+# frozen_string_literal: true
+
+require "helper"
+require "active_support/testing/isolation"
+
+class RailtieTest < ActiveSupport::TestCase
+  include ActiveSupport::Testing::Isolation
+
+  setup do
+    require "rails"
+
+    rails_logger = Logger.new(nil)
+
+    @app ||= Object.const_set(
+      "AJRailtieTestApp#{SecureRandom.alphanumeric(6)}",
+      Class.new(::Rails::Application)
+    ) do
+      config.eager_load = false
+      config.logger = rails_logger
```

Comment:
```suggestion
      config.logger = rails_logger
      config.active_support.cache_format_version = 7.1
```

Fixes a deprecation warning, see https://github.com/rails/rails/pull/49031.